leasing/management/commands/attach_areas.py

```diff
@@ -22,28 +24,37 @@ class Command(BaseCommand):
     def handle(self, *args, **options):  # noqa: C901 TODO
         from auditlog.registry import auditlog
 
+        logging.basicConfig(level=logging.INFO, format="%(message)s", stream=sys.stdout)
+        log = logging.getLogger(__name__)
```

Comment:
It's quite common convention to have the log as a module level variable (capitalized as `LOG` since it's "global"), i.e. just below the imports. That's because usually there's many functions in the same module which would all use the same logger.

In this case it seems that everything here is done in a single big function and therefore you can avoid passing the log object around. However, that's not very good style to have that big functions and IMHO that code should be eventually refactored to several smaller functions. Therefore I'd put the LOG object as module global already so that it wouldn't have to be changed when doing that kind of refactoring at some point.

Reply:
Moved to global scope.

Reply:
Otherwise good, but the basicConfig call was in correct place.

Reply:
Moved basicConfig inside function.